dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -113,6 +119,90 @@
 
+    /**
+     *
+     * Do a search, parameter are received by post and returns the json with the search info and contentlet results
+     *
+     * Example call using curl:
+     * curl --location --request POST 'http://localhost:8080/api/content/_search' \
+     * --header 'Content-Type: application/json' \
+     * --data-raw '{
+     *      	 "query": "+structurename:webpagecontent",
+     *       	 "sort":"modDate",
+     *       	 "limit":20,
+     *       	 "offset":0,
+     *           "userId":"dotcms.org.1"
+     * }'
+     *
+     * @param request {@link HttpServletRequest} object
+     * @param response {@link HttpServletResponse} object
+     * @param searchForm {@link SearchForm}
+     * @return json array of objects. each object with inode and identifier
+     */
+    @POST
+    @Path("/_search")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response search(@Context HttpServletRequest request,
+                           @Context final HttpServletResponse response,
+                           final SearchForm searchForm) throws DotSecurityException, DotDataException {
+
+        final InitDataObject initData = this.webResource.init
+                (null, request, response, false, null);
+
+        final User   user    = initData.getUser();
+        final String query   = searchForm.getQuery(); // ""
+        final String sort    = searchForm.getSort(); // ""
+        final int    limit   = searchForm.getLimit();  // 20
+        final int    offset  = searchForm.getOffset(); // 0
+        final PageMode pageMode   = PageMode.get(request);
+        final String userToPullID = searchForm.getUserId();
+        final String tmDate = (String) request.getSession().getAttribute("tm_date");
+        User   userForPull        = user;
+        List<ContentletSearch> contentletSearches = Collections.emptyList();
+        List<Contentlet>       contentlets        = Collections.emptyList();
+        long startAPISearchPull = 0;
+        long afterAPISearchPull = 0;
+        long startAPIPull = 0;
+        long afterAPIPull = 0;
+
+        Logger.debug(this, ()-> "Searching contentlets by: " + searchForm);
+
+        if(null != user && user.isAdmin()){
+
+            if(UtilMethods.isSet(userToPullID)) {
+
+                userForPull = APILocator.getUserAPI().loadUserById(userToPullID, APILocator.systemUser(),true);
+            }
+        }
+
+        if (UtilMethods.isSet(query)) {
+
+            startAPISearchPull = Calendar.getInstance().getTimeInMillis();
+            contentletSearches = APILocator.getContentletAPI().searchIndex(query, limit, offset, sort, userForPull, pageMode.respectAnonPerms);
+            afterAPISearchPull = Calendar.getInstance().getTimeInMillis();
+
+            startAPIPull       = Calendar.getInstance().getTimeInMillis();
+            contentlets        = ContentUtils.pull(processQuery(query), offset, limit, sort, user, tmDate, pageMode.respectAnonPerms)
```

Comment:
This needs to be run through the getJSON method so the results will match the existing api
```
                result = getJSON(contentlets, request, response, render, user, depth,
                        respectFrontendRoles, language, live, allCategoriesInfo);
```